lib/project_types/script/layers/application/project_dependencies.rb

```diff
@@ -9,13 +9,15 @@ def self.bootstrap(ctx:, language:, extension_point:, script_name:)
 
         def self.install(ctx:, language:, extension_point:, script_name:)
           dep_manager = Infrastructure::DependencyManager.for(ctx, language, extension_point, script_name)
-
           unless CLI::UI::Frame.open(ctx.message('script.project_deps.checking_with_npm')) do
-            return ctx.puts(ctx.message('script.project_deps.none_required')) if dep_manager.installed?
             begin
-              UI::StrictSpinner.spin(ctx.message('script.project_deps.installing')) do |spinner|
-                dep_manager.install
-                spinner.update_title(ctx.message('script.project_deps.installed'))
+              if dep_manager.installed?
+                ctx.puts(ctx.message('script.project_deps.none_required'))
+              else
+                UI::StrictSpinner.spin(ctx.message('script.project_deps.installing')) do |spinner|
+                  dep_manager.install
+                  spinner.update_title(ctx.message('script.project_deps.installed'))
+                end
```

Comment:
Not really introduced in this PR.  I am wondering about line 30 below.  Why is it raising `Infrastructure::Errors::DependencyInstallError`?  Should we just re-raise `e` in line 27 instead of returning false?